ctapipe/io/unofficial/targetio/camera.py

```diff
@@ -0,0 +1,195 @@
+"""
+Module for handling the camera specific information (such as pixel positions),
+which differ between CHEC-S, CHEC-M and single module files.
+"""
+import numpy as np
+from astropy import log
+from os.path import dirname, realpath, join
+from target_io import T_SAMPLES_PER_WAVEFORM_BLOCK as N_BLOCKSAMPLES
+from target_calib import MappingModule, MappingCHEC
+
+
+class Borg:
```

Comment:
What is `Borg`?

Reply:
So this is a class that essentially makes the Config a global object. Where-ever a new version of Config is created, it has the same settings as the others.

Its admittedly a horrible approach. But we have 2 prototype cameras, with 2 different configuration, with no way to tell them apart from the file alone. This allowed me to easily change the config for particular scripts without changing anything inside `TargetioFileReader`.

Reply:
But as with anything global, it sets off alarm bells because of traceability. I was hoping as it only really affects pixel positions, it will only be fiddled with by CHEC people, and it only exists temporarily as we transition to our newer camera, that it could be accepted. If not, then I'll have to change it :(

Reply:
Is the name a reference to Star Trek? Or is it an abbreviation, which is unknown to me? 
If it is a global config object, maybe one could call it `GlobalConfigObject` or so?

Reply:
Not sure, but probably. Its talked about on various blogs/stackexchange and I copied it:
http://code.activestate.com/recipes/66531-singleton-we-dont-need-no-stinkin-singleton-the-bo/
https://www.safaribooksonline.com/library/view/python-cookbook/0596001673/ch05s23.html

Reply:
I'd still recommend to use super() to its __init__

Reply:
Right now, this feels a bit like surprise to me. You can call an empty c-tor of `Config` and get a perfectly healthy instance. 

Somebody changes Config somewhere and you get a modified Config without even knowing it.

I assume this is exactly what you need here, for some reason. But for me this sounds pretty dangerous and difficult to understand.